index.bs

```diff
@@ -60,45 +58,41 @@ which redirects to <code>third.example</code>, which redirects to
 having tracking capabilities, the above behavior can result in Safari classifying
 <code>second.example</code> as having cross-site tracking capabilities.
 
-Safari deploys defenses against navigational-tracking if a navigation matches
+Safari deploys defenses against navigational tracking if a navigation matches
 all of the following criteria:
 
-<ul>
-  <li>The [=host/registrable domain=] that the user is navigating <em>from</em>
-      has been classified as having cross-site tracking capabilities</li>
-  <li>The URL the user is navigating <em>to</em> has either query parameters
-      or a URL fragment</li>
-</ul>
+* The [=host/registrable domain=] that the user is navigating <em>from</em>
+    has been classified as having cross-site tracking capabilities
+* The URL the user is navigating <em>to</em> has either query parameters
+    or a URL fragment
 
 When the above criteria holds the following defenses are deployed:
 
-<ul>
-  <li>The lifetime of client-side set cookies on the <em>destination page</em>
-      is capped at 24 hours.</li>
-  <li>If the user does not interact (i.e., register a user activation) on the
-      destination site within seven days of browser use, all non-cookie storage
-      is deleted.</li>
-</ul>
+* The lifetime of client-side set cookies on the <em>destination page</em>
+    is capped at 24 hours.
+* If the user does not interact (i.e., register a user activation) on the
+    destination site within seven days of browser use, all non-cookie storage
+    is deleted.
```

Comment:
The second bullet (non-cookie storage) is unconditional nowadays so no potential link decoration tracking needs to happen first.

Reply:
Okie ive pushed an update for this.  Can you take a look when you have a moment @johnwilander and let me know if i've described things correctly?